start.js

```diff
@@ -613,7 +611,9 @@ const exitProcess = () => {
 };
 
 readlineModule.emitKeypressEvents(process.stdin);
-process.stdin.setRawMode(true);
+if (process.stdin.isTTY) {
```

Comment:
So this little check makes it work for your Pi?

Reply:
yes. otherwise it can't run in the background via pm2